### PR TITLE
change links from personal repo to gitpod org

### PR DESCRIPTION
**Pull request by JesterOrNot**
Originally I kept all of these repositories on my personal account but to make the guide feel more official I changed the links to a gitpod-io org fork of the repositories. Also, I applied a few patches in regards to broken links.

### CODE REVIEW

File: src/docs/languages/bash.md [hunk 1]
```diff
@@ -30,7 +30,7 @@ USER gitpod
 RUN brew install shellcheck
 ```
 
-You should also install the shellcheck VS Code extension in Gitpod, by adding a [.gitpod.yml](https://www.gitpod.io/docs/41_config_gitpod_file/) configuration file to your repository that looks like this (notice the `vscode` extensions section):
+You should also install the shellcheck VS Code extension in Gitpod, by adding a [.gitpod.yml](https://www.gitpod.io/docs/config-gitpod-file/) configuration file to your repository that looks like this (notice the `vscode` extensions section):
```
Comment (nisarhassan12) [inline src/docs/languages/bash.md:33]: Thanks. Please remove  `https://www.gitpod.io` from the URL as with this in place it opens the link in the new tab which is not the what we want.

File: src/docs/languages/bash.md [hunk 2]
```diff
@@ -58,7 +58,7 @@ RUN npm i -g bash-language-server
 ```
 
 Also the following in your
-[.gitpod.yml](https://www.gitpod.io/docs/41-config-gitpod-file/)
+[.gitpod.yml](https://www.gitpod.io/docs/config-gitpod-file/)
```
Comment (nisarhassan12) [inline src/docs/languages/bash.md:61]: Do the same here and elsewhere:
image

File: src/docs/languages/python.md [hunk 3]
```diff
@@ -11,7 +11,7 @@ Before we get started, here are some examples of already-[gitpodified](https://w
...
 -----------|-----------------------------------------------------------|----------------------------------------------------
 | [gitpod-io/django-locallibrary-tutorial](https://github.com/gitpod-io/django-locallibrary-tutorial) | An example website written in Django by MDN |        [![Open in Gitpod](https://gitpod.io/button/open-in-gitpod.svg)](https://gitpod.io/#gitpod-io/django-locallibrary-tutorial) |
-| [JesterOrNot/Gitpod-PyQt](https://github.com/JesterOrNot/Gitpod-PyQt)                               |    A PyQt example for Gitpod | [![Open in Gitpod](https://gitpod.io/button/open-in-gitpod.svg)](https://gitpod.io/#https://github.com/JesterOrNot/Gitpod-PyQt) |
+| [gitpod-io/Gitpod-PyQt](https://github.com/gitpod-io/Gitpod-PyQt)                               |    A PyQt example for Gitpod | [![Open in Gitpod](https://gitpod.io/button/open-in-gitpod.svg)](https://gitpod.io/#https://github.com/JesterOrNot/Gitpod-PyQt) |
```
Comment (nisarhassan12) [inline src/docs/languages/python.md:14]: Many! Thanks! This is still pointing to your account maybe that's becuase It's un-official right :slightly_smiling_face: ?

Comment (nisarhassan12) [reply src/docs/languages/python.md:14]: image

Comment (JesterOrNot) [reply src/docs/languages/python.md:14]: Good catch!

Comment (JesterOrNot) [reply src/docs/languages/python.md]: Done!

**Comment by JesterOrNot** [comment]
Thanks, Merging! :rocket:

**Pull request merged**